packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js

```diff
@@ -90,35 +92,15 @@ describe( 'getKarmaConfig()', () => {
 
 		expect( karmaConfig ).to.have.own.property( 'proxies' );
 		expect( karmaConfig.proxies ).to.have.own.property( '/assets/' );
+		expect( karmaConfig.proxies ).to.have.own.property( '/example.com/image.png' );
+		expect( karmaConfig.proxies ).to.have.own.property( '/www.example.com/image.png' );
 
 		expect( karmaConfig.files ).to.be.an( 'array' );
 		expect( karmaConfig.files.length ).to.equal( 2 );
 		expect( karmaConfig.files[ 0 ] ).to.equal( 'workspace/entry-file.js' );
 		expect( karmaConfig.files[ 1 ].pattern ).to.equal( 'packages/ckeditor5-utils/tests/_assets/**/*' );
 	} );
 
-	// See: https://github.com/ckeditor/ckeditor5/issues/8823
-	it( 'should define proxies to static assets resources', () => {
```

Comment:
It isn't required anymore, right?

Reply:
There were two tests with exactly the same description that checked almost the same things. I joined their `expect()` together inside one test.